### DISCUSSION TITLE
Romeo? It doesn't seem to be used anymore.

### DISCUSSION THREAD

**Discussion by carlosmintfan** [linuxmint/discussions#145] [Development] [answered]
If the romeo package channel isn't used anymore:
1. Remove empty romeo section from https://packages.linuxmint.com
2. Replace romeo toggle with daily builds PPA toggle in mintSources
3. In mintUpdate, don't show romeo packages as unstable:
```python
                    if origin.origin == "linuxmint":
                        if origin.component == "romeo":
                            self.type = "unstable"
``` 
https://github.com/linuxmint/mintupdate/blob/master/usr/lib/linuxmint/mintUpdate/Classes.py#L129
but instead updates from the daily builds PPA:
https://github.com/linuxmint/mintupdate/blob/master/usr/lib/linuxmint/mintUpdate/Classes.py#L117
Add this:
```python
                    elif origin.origin.startswith("LP-PPA"):
                        self.origin = origin.origin
                        if origin.origin == "linuxmint-daily-build-team-daily-builds":
                            self.type = "unstable"
                            break
```

**Comment by clefebvre** [comment, marked as answer]
Romeo is used. It's where we stage unstable packages to test them.

You don't see anything in there because this usually happens fast and we remove them from Romeo when they land in the stable components.
